src/containers/pages/JurisdictionAssignment/AutoSelectJurisdictions/tests/e2e.test.tsx

```diff
@@ -0,0 +1,300 @@
+    const tbodyRow = wrapper.find('tbody tr');
+    expect(tbodyRow.length).toEqual(1);
+    expect(
+      tbodyRow
+        .at(0)
+        .text()
+        .includes('Zambia')
+    ).toBeTruthy();
+    // expect only one tbody row
+    expect(wrapper.find('tbody tr').length).toEqual(1);
+
+    expect(
+      wrapper
+        .find('tbody tr')
+        .at(0)
+        .text()
+    ).toMatchInlineSnapshot(`"Zambia178"`);
+
+    // test drilldown
+    tbodyRow
+      .at(0)
+      .find('NodeCell Link a')
+      .simulate('click', { button: 0 });
+
+    wrapper.update();
+    renderTable(wrapper, 'after first click');
+
+    // expect only one tbody row
+    expect(wrapper.find('tbody tr').length).toEqual(1);
+    expect(
+      wrapper
+        .find('tbody tr')
+        .at(0)
+        .text()
+    ).toMatchInlineSnapshot(`"Eastern178"`);
+
+    wrapper
+      .find('tbody tr')
+      .at(0)
+      .find('NodeCell a')
+      .simulate('click', { button: 0 });
+
+    wrapper.update();
+    renderTable(wrapper, 'after second click');
+
+    // expect only one tbody row
+    expect(wrapper.find('tbody tr').length).toEqual(1);
+    expect(
+      wrapper
+        .find('tbody tr')
+        .at(0)
+        .text()
+    ).toMatchInlineSnapshot(`"Sinda178"`);
+
+    wrapper
+      .find('tbody tr')
+      .at(0)
+      .find('NodeCell a')
+      .simulate('click', { button: 0 });
+
+    wrapper.update();
+    renderTable(wrapper, 'after third click');
+
+    // expect only one tbody row
+    expect(wrapper.find('tbody tr').length).toEqual(1);
+    expect(
+      wrapper
+        .find('tbody tr')
+        .at(0)
+        .text()
+    ).toMatchInlineSnapshot(`"Chiwuyu178"`);
+
+    wrapper
+      .find('tbody tr')
+      .at(0)
+      .find('NodeCell a')
+      .simulate('click', { button: 0 });
+
+    wrapper.update();
+    renderTable(wrapper, 'after third click');
+
+    // expect only one tbody row
+    expect(wrapper.find('tbody tr').length).toEqual(1);
+    expect(
+      wrapper
+        .find('tbody tr')
+        .at(0)
+        .text()
+    ).toMatchInlineSnapshot(`"CWY_25178"`);
+
+    // CWY_25 is the last child it should not be clickable
+    expect(toJson(wrapper.find('tbody tr NodeCell span'))).toMatchSnapshot(
+      'should not have btn-link or onClick handler'
+    );
+
+    /** try resource calculation for CWY_25 */
+
+    // simulate structure count change so that we can have 10 structures per team
+    wrapper
+      .find('input[name="structuresCount"]')
+      .simulate('change', { target: { value: 10, name: 'structuresCount' } });
+    wrapper.update();
+
+    wrapper
+      .find('input[name="teamsCount"]')
+      .simulate('change', { target: { value: 3, name: 'teamsCount' } });
+    wrapper.update();
+
+    // see if there has been any change to the number of days, should start with 6 days
+    // 178 structures / 30 structures per team for 3 teams => 178 / 30
+    expect(wrapper.find('form p').text()).toMatchInlineSnapshot(
```

Comment:
Lets add a test here for what happens when there are no jurisdictions selected

Reply:
cool.